### PR TITLE
Improve Bluetooth docs

### PR DESCRIPTION
**Pull request by SKHSKHilber**
Please ignore.

I tried to suggest 2 changes to make things clearer in 

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/builders/installation.md#install-phoniebox-software 

and 

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/builders/audio.md

not 787 changes !

### CODE REVIEW

**Comment by AlvinSchiller** [comment]
You just chose the wrong target branch. I changed it to `future3/main`.

But could you please do your changes on a `future3/develop` based branch and also choose this as the target branch?

You could reset your branch on it and cherry pick your changes if you are familiar enough with git.

**Comment by s-martin** [comment]
@AlvinSchiller its now on future3/main, did you want it on future3/develop instead?

**Comment by AlvinSchiller** [comment]
> @AlvinSchiller its now on future3/main, did you want it on future3/develop instead?

Not yet, as the PR branch is based on `future3/main`. This way only the real changes are shown.
If the branch is corrected it can be changed to `future3/develop`.

File: documentation/builders/audio.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ Bluetooth setup consists of three steps
 
 To pair and connect, follow these steps. This will be a one-time setup.
 
-```bash
+```
+sudo bluetoothctl
+bash
```
Comment (pabera) [inline documentation/builders/audio.md:68]: I think there is a small mistake here..

it should be

```
```bash
sudo bluetoothctl
...
```

**Review by s-martin — APPROVED** [review]

**Comment by s-martin** [comment]
@AlvinSchiller we need that on dev from my point of view

**Comment by pabera** [comment]
Just changed it to point to `future3/develop`

**Comment by AlvinSchiller** [comment]
> Just changed it to point to future3/develop

thats not enough, as the head branch is based on main. Thats why i left the base for now.
I rebase the branch to develop now

**Review by AlvinSchiller — APPROVED** [review]

**Comment by AlvinSchiller** [comment]
It worked for me without sudo btw

**Comment by s-martin** [comment]
> It worked for me without sudo btw. tested on bullseye and bookworm.

so we close the pr?

**Comment by AlvinSchiller** [comment]
@SKHSKHilber I added your second suggestion about the 64-bit kernel fix in https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2216

**Pull request merged**